### ISSUE TITLE
Problem with getting accountId checksum 

### ISSUE THREAD

**Issue by soharang** [hashgraph/hedera-sdk-swift#38]
### Description

# What happened
Cannot get checksum for account id and the following error occurred.
![image](https://user-images.githubusercontent.com/32191478/220011388-cc904a6b-eb67-46a0-b816-11e8fb405e4d.png)
<img width="1070" alt="image" src="https://user-images.githubusercontent.com/32191478/220011458-20fa533a-cc0e-480c-b19a-b8190862a592.png">

# What Expected
It should return checksum address for given accout.

### Steps to reproduce

1. Import Hedera swift sdk with Package manager.

2. Run unit test with following codes.
```
func testAccountChecksumTest() throws {
    let checksumed = AccountId("0.0.1126123").toStringWithChecksum(.forMainnet())
    print("hedera checksum=\(checksumed)")
}
```

### Additional context

I tried following codes in hedera-sdk-swift with EntityIdTests.swift, which is almost same as above codes.
```
internal func testAccountIdChecksum() {
    let checksumAddress = AccountId("0.0.1126123").toStringWithChecksum(.forMainnet())
    print("testAccountIdChecksum(): checksumed=\(checksumAddress)")
}
```
By the way, it always returns values for Testnet.
<img width="950" alt="image" src="https://user-images.githubusercontent.com/32191478/220012160-175b4206-65eb-4870-9969-ad0fe71e2573.png">

I wonder if I tried right approach for getting account checksum.

### Hedera network

mainnet

### Version

v0.13.0

### Operating system

macOS

**Comment by izik1** [comment]
I believe this will be fixed by hashgraph/hedera-sdk-rust#449

**Comment by soharang** [comment]
@izik1  Thanks for your comment.
Do you think I need to create another report for my additional context? - It always returns checksum for the testnet.

**Comment by izik1** [comment]
That one should've already been fixed in https://github.com/hashgraph/hedera-sdk-rust/commit/ed55bbb6a6c002be93018abc9df31b6f663bdb92, I can't reproduce it locally as of hashgraph/hedera-sdk-rust#449

**Comment by soharang** [comment]
@izik1 Thank you! It is so clear.

**Comment by izik1** [comment]
This should be fixed as of the latest release (0.14), if it isn't feel free to re-open the issue.

**Comment by soharang** [comment]
@izik1 
Thank you. I got new sdk and the problems are solved clearly.
